### PR TITLE
3350 variant index

### PR DESCRIPTION
**Pull request by remo87**

### CODE REVIEW

File: terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv [hunk 1]
```diff
@@ -32,3 +32,5 @@ fda/significantAdverseDrugReactions,openfda_faers,,default.json
 pharmacogenomics,pharmacogenomics,,default.json
 facetSearchTarget,facet_search_target,,facet_search.json
 facetSearchDisease,facet_search_disease,,facet_search.json
+variantIndex,variant_index,variantId,default.json
```
Comment (mbdebian) [inline terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv:35]: Isn't it using the new settings 'variant_index.json' included in this PR?

**Review by mbdebian — CHANGES_REQUESTED** [review]
Good progress with this PR @remo87 !
I may be missing something about the new _variant index_ in opensearch, I just left a comment in the PR.
Thanks!

File: terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv (hunk 1 above)
Comment (remo87) [reply terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv:35]: sorry forgot to remove the index configuration I thought I was going to need it and in the end the default worked just fine

File: terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv
[the remark this answers is not in the document]
Comment (remo87) [reply terraform_create_images/scripts/posvm/postproduction/elasticsearch/metadata/etl_ingest_configuration.csv:1]: sorry forgot to remove the index configuration I thought I was going to need it and in the end the default worked just fine

**Review by mbdebian — APPROVED** [review]
Thanks @remo87 !

**Pull request merged**